src/common.speech/SynthesisAdapterBase.ts

```diff
@@ -356,24 +359,59 @@ export class SynthesisAdapterBase implements IDisposable {
                     case "audio.metadata":
                         const metadataList = SynthesisAudioMetadata.fromJSON(connectionMessage.textBody).Metadata;
                         for (const metadata of metadataList) {
-                            if (metadata.Type.toLowerCase() === "WordBoundary".toLowerCase()) {
-
-                                this.privSynthesisTurn.onWordBoundaryEvent(metadata.Data.text.Text);
-
-                                const ev: SpeechSynthesisWordBoundaryEventArgs = new SpeechSynthesisWordBoundaryEventArgs(
-                                    metadata.Data.Offset,
-                                    metadata.Data.text.Text,
-                                    metadata.Data.text.Length,
-                                    this.privSynthesisTurn.currentTextOffset);
-
-                                if (!!this.privSpeechSynthesizer.wordBoundary) {
-                                    try {
-                                        this.privSpeechSynthesizer.wordBoundary(this.privSpeechSynthesizer, ev);
-                                    } catch (error) {
-                                        // Not going to let errors in the event handler
-                                        // trip things up.
+                            switch (metadata.Type) {
+                                case MetadataType.WordBoundary:
+                                    this.privSynthesisTurn.onWordBoundaryEvent(metadata.Data.text.Text);
+
+                                    const ev: SpeechSynthesisWordBoundaryEventArgs = new SpeechSynthesisWordBoundaryEventArgs(
```

Comment:
could "ev," "bev," and "vev" have more descriptive names, like "boundaryEventArgs," "bookmarkEventArgs," and "visemeEventArgs"?

Reply:
thx, updated them